scripts/test.js

```diff
@@ -112,4 +113,16 @@ executor.addCommandlineTest(
       message: 'warning -- Module "xApp" was created but never loaded.'
     }]);
 
+// Check accessibility plugin
+
+executor.addCommandlineTest(
+    'node lib/cli.js plugins/accessibility/spec/failureConfig.js')
+    .expectExitCode(1)
+    .expectErrors([{
+      message: 'warning -- accessibility plugin cannot be run as there is ' +
```

Comment:
I'm not sure we need this test, since it's not adding anything to the plugin test above.

Could we instead have failureConfig.js load up a file that has accessibility errors, and then assert that they're output?